### PR TITLE
PT-BR Translation

### PR DESCRIPTION
**Pull request by FernandoJackMedeiros**
White Paper revisado em português-Br

### CODE REVIEW

**Comment by canesin** [comment]
Thanks @FernandoJackMedeiros, @fredafrica and @erikzhang I will take care of this as Fernando is one of my puppies... 

Jack, continua adicionando ai, não esquece do sumário. Arquivo "toc" .. me manda um DM no slack depois.

File: en-us/node/consensus.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ For the sake of discussion, we will describe a couple scenarios.  In these simpl
 The practical implementation of DBFT in NEO uses an iterative consensus method to guarantee that consensus is reached.  The performance of the algorithm is dependent on the fraction of honest nodes in the system.**Figure 5** depicts the
 expected iterations as a function of the fraction of dishonest nodes.  
 
-Note that the **Figure 5** does not extend below 66.66% **Consensus Node** honesty.  Between this critical point and 33% **Consensus Node** honesty, there is a 'No-Man's Land' where a consensus is unattainable.  Below 33.33% **Consensus Node** honesty, dishonest nodes (assuming they are aligned in consensus) are able to reach a consensus themselves and become the new point of truth in the system.
+Note that the **Figure 5** does not extend below 66.66% **Delegates** honesty.  Between this critical point and 33% **Delegates** honesty, there is a 'No-Man's Land' where a consensus is unattainable.  Below 33.33% **Delegates** honesty, dishonest nodes (assuming they are aligned in consensus) are able to reach a consensus themselves and become the new point of truth in the system.
```
Comment (canesin) [inline en-us/node/consensus.md]: As a new language addition you should not change files outside the language folder (bisides toc). Please discart changes to en-us, do another PR if you want those.

Comment (FernandoJackMedeiros) [reply en-us/node/consensus.md]: okk

**Comment by canesin** [comment]
@shakilmia are you ok ? why you keep doing this ?

**Comment by canesin** [comment]
@FernandoJackMedeiros I'm accepting to put it out for a first revision.
